UnityProject/Assets/Scripts/GameModes/Extended.asset

```diff
@@ -14,6 +14,7 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   Name: Extended
   Description: Relax on the station without any pesky antags bothering you.
-  CanRespawn: 0
+  CanRespawn: 1
```

Comment:
I think it’s best to modify the “respawn local player” menu item to force respawn instead of having to tick these all the time

Reply:
Oh sorry, I actually didn't mean to check this in